flask_common/utils.py

```diff
@@ -49,19 +47,24 @@ def json_list_generator(results):
             break
     yield ']'
 
-class isortedset(sortedset):
-    def __init__(self, *args, **kwargs):
-        if not kwargs.get('key'):
-            kwargs['key'] = lambda s: s.lower()
-        super(isortedset, self).__init__(*args, **kwargs)
-
-    def __contains__(self, key):
-        if not self:
-            return False
-        try:
-            return self[self.bisect_left(key)].lower() == key.lower()
-        except IndexError:
-            return False
+try:
+    from blist import sortedset
```

Comment:
You can only do `from flask_common.utils import isortedset` if `blist` is installed. Previously you wouldn't be able to import the entire module. You're not supposed to import `flask_common.utils.sortedset` (even though you can).